server/src/main/java/io/crate/execution/engine/aggregation/impl/PercentileAggregation.java

```diff
@@ -107,6 +137,10 @@ public TDigestState iterate(RamAccounting ramAccounting,
                                 Input<?>... args) throws CircuitBreakingException {
         if (state.isEmpty()) {
             Object fractionValue = args[1].value();
+            if (args.length > 2) {
```

Comment:
```suggestion
            if (args.length == 3) {
```

Minor: This is more precise.

Reply:
But would need adjusted if one would add another argument. I'd leave it like that.